src/v2/BaseLoginRouter.js

```diff
@@ -184,10 +184,11 @@ export default Router.extend({
       return this.handleUpdateAppState(error.details);
     }
 
-    // assume it's a config error
-    this.settings.callGlobalError(new Errors.ConfigError(
-      error
-    ));
+    // If the error is a string, wrap it in an Error object
+    if (typeof error === 'string') {
+      error = new Error(error);
+    }
+    this.settings.callGlobalError(error);
```

Comment:
Looks like this change adds normal `Error` instead of `Errors.ConfigError` to the globalError, intentional change?

Reply:
Yes this "assume it is config error" was creating bad error stacks and obscuring the original message. I searched code in monolith and widget and issues on GH, I believe we can make this fix without negative effect